packages/server/graphql/types/CreatePollPayload.ts

```diff
@@ -0,0 +1,21 @@
+import {GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import makeMutationPayload from './makeMutationPayload'
+import Poll from './../types/Poll'
+
+export const CreatePollSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'CreatePollSuccess',
+  fields: () => ({
+    poll: {
```

Comment:
nit: sometimes i like to include the field `pollId` in case a future use case only needs the Id instead of fetching the whole poll

Reply:
Just to make sure, with a format of `poll:idGeneratedByDatabase`, not raw id generated by DB? 😅

Reply:
This was a dumb question, it's a resolver so can't be anything else but `poll:idGeneratedByDatabase` to keep things consistent.